domain/status/state/state.go

```diff
@@ -40,6 +41,40 @@ func NewState(factory database.TxnRunnerFactory, clock clock.Clock, logger logge
 	}
 }
 
+// GetModelInfo returns the model's basic information.
+func (st *State) GetModelInfo(ctx context.Context) (status.ModelStatusInfo, error) {
+	type modelInfo struct {
+		ModelType       string `db:"type"`
+		CredentialOwner string `db:"credential_owner"`
```

Comment:
Credential owner is not the same as model owner. Unfortunately in this context we do not have the model owner currently available. We have plans on what this might look like and how it will change. For the time can I suggest just putting a TODO around getting the model owner and setting the value in the status api to an empty string?

Reply:
I see, sure and thanks tom!